test/model/model_test.rb

```diff
@@ -112,6 +112,14 @@ class Event
       assert_equal nil, @event.location
     end
 
+    test 'reload object with given quorum' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.